CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_reddit_ads v1.0.1
+
+[PR #27](https://github.com/fivetran/dbt_reddit_ads/pull/27) includes the following updates:
+
+## Schema Updates
+
+**10 total changes • 0 possible breaking changes**
+
+| Data Model(s) | Change type | Old | New | Notes |
+| ---------- | ----------- | -------- | -------- | ----- |
+| [stg_reddit_ads__account_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__account_conversions_report)<br>[stg_reddit_ads__ad_group_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__ad_group_conversions_report)<br>[stg_reddit_ads__ad_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__ad_conversions_report)<br>[stg_reddit_ads__campaign_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__campaign_conversions_report)<br>[stg_reddit_ads__campaign_country_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__campaign_country_conversions_report) | New Columns |  | `clicks`<br>`views` | Added to support API v3. Coalesced downstream with the deprecated columns `click_through_conversion_attribution_window_month` and `view_through_conversion_attribution_window_month`. |
+| [stg_reddit_ads__account_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__account_conversions_report)<br>[stg_reddit_ads__ad_group_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__ad_group_conversions_report)<br>[stg_reddit_ads__ad_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__ad_conversions_report)<br>[stg_reddit_ads__campaign_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__campaign_conversions_report)<br>[stg_reddit_ads__campaign_country_conversions_report_tmp](https://fivetran.github.io/dbt_reddit_ads/#!/model/model.reddit_ads.stg_reddit_ads__campaign_country_conversions_report) | Deprecated Columns | `click_through_conversion_attribution_window_month`<br>`view_through_conversion_attribution_window_month` |  | Coalesced downstream with the new columns `clicks` and `views`. |
```

Comment:
I'd prefer we make the `Note` from the first section regarding the new columns to be it's own section for the non tmp models. My recommendation would be to have a new row for the non tmp staging models after the callout of the new fields in the tmp models and highlight the coalesce modification to the fields in the staging model.